src/docs/metrics/contract-interacting-address-count/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Interacting Address Count
+author: Yury
+date: 2022-04-21
+description: Get interacting address count for a chain address / contract
+---
+
+## Definition
+
+Get interacting address count for a chain address / contract
+
+[Example of usage **interacting address count** on Sanbase](https://app.santiment.net/s/rk-8It-D)
```

Comment:
I don't think it's a good idea to add links to personal chart layouts. We also don't do this in other places.
Furthermore, these metrics are *very* slow to compute, so even if you click - nothing happens for long period of time.

In this case, I think screenshots would work better. And if we have link to charts, they should not be to personal stuff but better to be a short URL without a chart layout.

Reply:
@IvanIvanoff You are right. Thank you. Will update it.